pootle/apps/pootle_app/forms.py

```diff
@@ -64,13 +64,14 @@ def __init__(self, *args, **kwargs):
             ).count
 
             if has_stores:
-                #self.fields['localfiletype'].widget.attrs['disabled'] = True
+                # self.fields['localfiletype'].widget.attrs['disabled'] = True
```

Comment:
Likely that's a better approach.  But we're left with cases where temporary code is wanting to be commented out.  We could decide to ignore this check.